src/appengine/handlers/base_handler_flask.py

```diff
@@ -203,7 +203,8 @@ def render(self, path, values=None, status=200):
 
     template = _JINJA_ENVIRONMENT.get_template(path)
 
-    response = Response()
+    if not response:
+      response = Response()
```

Comment:
possibly dumb question: can `Response()` be used as a default value (on line 184)?

Reply:
I don't think we can, since that instance would be shared for all invocations.